### PR TITLE
Gaming page enhancements and fixes

### PR DESCRIPTION
**Pull request by MichaIng**
- ~Make FAQ own tabs.~ @StephanStS was faster 😉.
- Align access data like URL, IP, passwords etc.
- Minor spelling and wording + start new headers and bullet points consequently with a capital letter but do not use capital letters for subsequent words in headers, if those are not names.
- If a console command or file path starts at a new line already, wrap it into fenced code syntax, which adds a copy button, allows syntax highlighting and looks cleaner overall.
- PaperMC: Add a new tab about memory management and how to change the Java heap size, which is an important parameter to run the server reliably.
- Give image alternative tags more meaning when read by website TTS engines.

@Joulinar 
I added a section to PaperMC about the Java memory issues, how to change the heap size and a warning about having a regularly used swap file on SD card. We can print an additional warning during PaperMC install and refer to this section, as it's mostly mandatory to adjust the value.

### CODE REVIEW

**Comment by MichaIng** [comment]
I have no idea why GitHub has issues to merge branches with commits from the desktop client, even that conflicts have been resolved and all commits merged.

**Review by StephanStS — APPROVED** [review]

**Comment by MichaIng** [comment]
We have so many keyboard key combinations here, a good reason to start implementing the dedicated Markdown plugin for those 😃.

File: mkdocs.yml [hunk 1]
```diff
@@ -41,13 +41,12 @@ markdown_extensions:
...
-  - pymdownx.superfences # Indended code blocks, e.g. within lists, admonition, details or tabbed blocks
-  - pymdownx.tabbed # Tabbed content blocks
-  - pymdownx.tasklist: # Task list support
-      custom_checkbox: true # Custom icons
+  - pymdownx.keys # Keyboard key symbols
   - pymdownx.magiclink: # Auto-links non angle bracket URLs and email addresses and allows to shorten and visualise Git repo links
       repo_url_shortener: true
       repo_url_shorthand: true
+  - pymdownx.superfences # Indended code blocks, e.g. within lists, admonition, details or tabbed blocks
+  - pymdownx.tabbed # Tabbed content blocks
```
Comment (MichaIng) [inline mkdocs.yml:49]: Strange, @fpetru do you have any idea why the above changes broke MkDocs parsing? https://github.com/MichaIng/DietPi-Docs/runs/1703693561?check_suite_focus=true
(The two little indentation alignments were attempts by me to fix the issue.)
YAML validators report correct syntax, aside of the `!!python/name:` values which are special MkDocs syntax to run Python code, but those were there before.

I don't even understand the error message as there is no block starting at the first line, at least if you do not count a whole YAML file as block:
```
MkDocs encountered as error parsing the configuration file: while parsing a block mapping
  in "/home/runner/work/DietPi-Docs/DietPi-Docs/mkdocs.yml", line 1, column 1
expected <block end>, but found '<block sequence start>'
  in "/home/runner/work/DietPi-Docs/DietPi-Docs/mkdocs.yml", line 49, column 3
```

Comment (MichaIng) [reply mkdocs.yml:49]: Interesting, I use the exact same file (verified with `diff`) on our server and it builds the site without any issue 😕. Python version and all modules match, strange. Let's see if the complete branch builds fine.

**Comment by Joulinar** [comment]
Ok let me see where to place the warning within the installation process. Maybe I will add it to the information window informing about potential long installation time

**Comment by MichaIng** [comment]
Jep, that seems like a good place.

File: mkdocs.yml (hunk 1 above)
Comment (MichaIng) [reply mkdocs.yml]: That solved it: https://github.com/MichaIng/DietPi-Docs/pull/314/commits/3b6587b1ca8f1a50eaf7b797026b88d28085baa6
Mysterious, as locally it doesn't matter. I worry now that something could be parsed wrong, as correct syntax (the way it looks) produces an error.

**Comment by MichaIng** [comment]
Interesting, the latest spell check results reveal that the magiclink extension, or at least the URL shortener option did not work before within GitHub actions, while the live docs have successfully shortened URLs. Only explanation is the replaced repo_url and edit_url values, probably the variable expansion did not work as expected.

File: mkdocs.yml (hunk 1 above)
Comment (MichaIng) [reply mkdocs.yml]: Mystery solved, all working as expected now.

**Pull request merged**